_studio/mfx_lib/encode_hw/vp9/include/mfx_vp9_encode_hw_utils.h

```diff
@@ -274,7 +274,7 @@ enum // identifies memory type at encoder input w/o any details
     template<class T, class U> inline void Copy(T & dst, U const & src)
     {
         static_assert(sizeof(T) == sizeof(U), "copy objects of different size");
-        memcpy_s(&dst, sizeof(dst), &src, sizeof(dst));
+        dst = src;
```

Comment:
Types can be nonconvertible or implicit conversion may happen. It is not the same as memcpy. Would be better to delete this function.

Reply:
I agree, suggest to fully remove this function and replace with operator =